examples/async/index.js

```diff
@@ -1,15 +1,43 @@
 import 'babel-polyfill'
 import React from 'react'
-import { render } from 'react-dom'
+import ReactDOM from 'react-dom'
 import { Provider } from 'react-redux'
-import App from './containers/App'
 import configureStore from './store/configureStore'
 
 const store = configureStore()
+const rootEl = document.getElementById('root')
 
-render(
-  <Provider store={store}>
-    <App />
-  </Provider>,
-  document.getElementById('root')
-)
+let render = () => {
+  const App = require('./containers/App').default
```

Comment:
Yeah I'm also struggling to get this working in a real life application, ideally I'd just have to put the `module.hot.accept` somewhere "high up" (e.g. my entry point) and then changes would bubble up there and it would reload the whole thing, but I can't seem to get this working